modules/ding_borgun_checkout/ding_borgun_checkout.info

```diff
@@ -0,0 +1,5 @@
+name = Ding Borgun.Checkout
+description = Enable Borgun.Checkout payment of debts
```

Comment:
I think we need to add some module requirements here e.g.

- ding_provider
- libraries

Reply:
Added